### PR TITLE
add slack link in evaluate nexus 

### PR DESCRIPTION
**Pull request by jsundai**

### CODE REVIEW

File: docs/evaluate/development-production-features/temporal-nexus.mdx [hunk 1]
```diff
@@ -28,6 +28,9 @@ Temporal Nexus is available in [Public Preview](https://docs.temporal.io/evaluat
 Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
 It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
 
+For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/temporalnexus).
```
[the remark this answers is not in the document]
Comment (bergundy) [reply docs/evaluate/development-production-features/temporal-nexus.mdx:31]: Can we use `/nexus/concepts` or `/nexus/wiki`?

[the remark this answers is not in the document]
Comment (jsundai) [reply docs/evaluate/development-production-features/temporal-nexus.mdx:31]: We have /nexus for the evaluate page and /temporalnexus for the encyclopedia. cloud/nexus for cloud docs. We can't have /nexus for the encyclopedia but I can make it docs.temporal.io/encyclopedia/nexus ?

[the remark this answers is not in the document]
Comment (bergundy) [reply docs/evaluate/development-production-features/temporal-nexus.mdx:31]: As discussed on slack let's change `/temporalnexus*` to `/nexus*` and what's now at `/nexus` to `/evaluate/nexus`.

**Review by fairlydurable — APPROVED** [review]
Unblocking this PR for timely updates.

Questions to consider:


- The last part of the conversation was "As discussed on slack let's change /temporalnexus* to /nexus* and what's now at /nexus to /evaluate/nexus." and it is still /temporalnexus.  I do not see a comment that explains why the review feedback was marked as resolved without an explanation in the record as to why it was resolved.
- Why does one Slack link use the shortcut and one does not?
- Why is Encyclopedia capitalized in the link? Also, "entry" is more common than "section" for Encyclopedia content.

**Comment by jsundai** [comment]
Just to update, this was resolved through slack convos and a previous PR merged. The Slack invite link is one that we'll need to monitor in case it changes. 

Thank you, Erica for the approval on short notice! Hoping to get your eyes on this project later down the line once style reviews happen.  🙌

**Pull request merged**